scripts/release/version-packages.ts

```diff
@@ -50,6 +50,13 @@ async function versionPackages() {
     }
   }
 
+  if (releaseType === 'release-candidate') {
```

Comment:
This should probably go into `if-else` or a `switch` statement and throw on unknown release types.